src/main/java/org/openjdk/shenandoah/DataLogProvider.java

```diff
@@ -0,0 +1,228 @@
+/*
+ * ====
```

Comment:
This copyright notice looks wrong. Also, update copyright notices in all files that you have changed.

Reply:
Why are you adding Red Hat to files that you created new? Or have you moved existing code into those new files?

Reply:
I'll go over the copyright notices.